test/NVQPP/struct_arg.cpp

```diff
@@ -7,7 +7,7 @@
  ******************************************************************************/
 
 // RUN: nvq++ %cpp_std --emit-qir %s && cat struct_arg.qir.ll | \
-// RUN: FileCheck %s ; rm struct_arg.qir.ll
+// RUN: FileCheck %s && rm struct_arg.qir.ll
```

Comment:
Don't we always want to erase the intermediate file? Although we shouldn't return the return code of `rm`. heh